src/components/Search/Search.js

```diff
@@ -0,0 +1,105 @@
+import React, { useMemo, useState } from 'react';
+import Autosuggest from 'react-autosuggest';
+import {
+  FaSearch,
+  FaChevronCircleDown,
+  FaChevronCircleUp,
+} from 'react-icons/fa';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+
+import SearchService from './SearchService';
+import SearchToggle from './SearchToggle';
+
+import './Search.scss';
+
+function Search() {
+  const dispatch = useDispatch();
+  const activeTopic = useSelector(state => state.app.activeTopic);
+  const [suggestions, setSuggestions] = useState([]);
+  const [value, setValue] = useState('');
+  const { t } = useTranslation();
+
+  const searchService = useMemo(() => {
+    const clear = () => setSuggestions([]);
+    const upsert = (section, items) =>
+      setSuggestions(oldSuggestions => {
+        const index = oldSuggestions.findIndex(s => s.section === section);
+        const newSuggestions = [...oldSuggestions];
+        if (index === -1 && items) {
+          newSuggestions.push({ section, items });
+        } else if (items) {
+          newSuggestions[index] = { section, items };
+        }
+        return newSuggestions;
+      });
+    return new SearchService(activeTopic, clear, upsert, dispatch);
+  }, [activeTopic, setSuggestions, dispatch]);
+
+  return (
+    Object.keys(activeTopic.searches).length > 0 && (
+      <div className="wkp-search">
+        <SearchToggle>
+          <Autosuggest
+            multiSection
+            suggestions={suggestions}
```

Comment:
How does work the multiSection ? each suggestion has a section property?

Reply:
Suggestions are grouped by section (see https://codepen.io/moroshko/pen/qbRNjV for an example). To be more flexible I've also added the section property to the suggestion.

Reply:
Ah yes I didn't have the last version , grouping is very nice.